spec/imsc-hrm.html

```diff
@@ -823,14 +815,20 @@ <h2>Paint Text</h2>
       height)<sup>2</sup></p>
 
       <p class='note'>NRGA(G<sub>i</sub>) does not take into account decorations (e.g. underline), effects (e.g. outline) or actual
-      typographical glyph aspect ratio. An implementation can determine an actual buffer size needs based on worst-case glyph size
+      typographical glyph aspect ratio. An implementation can determine an actual cache size needs based on worst-case glyph size
       complexity.</p>
 
-      <p>The contents of the Glyph Buffer G<sub>n</sub> are copied instantaneously to Glyph Buffer G<sub>n-1</sub> at the
-      presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>.</p>
+      <p>At the presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>:</p>
+      <ol>
+        <li>purge from the Glyph Cache all <a>glyphs</a> not flagged <i>retain</i>; and</li>
+        <li>remove the <i>retain</i> flag from all remaining <a>glyphs</a> in the Glyph Cache; and</li>
+      </ol>
```

Comment:
Same comment as for image cache: the ordering of these steps is important.

Reply:
See https://github.com/w3c/imsc-hrm/pull/55/commits/8be035878a1943580b190e2187d43c0698ac22a3